build.gradle

```
@@ -340,7 +340,7 @@ task spigotResources(type: ProcessResources) {
 		include '**'
 		version = project.property('version')
 		def channel = 'stable'
-		if (version.contains('pre'))
+		if (version.contains('pre') || version.contains('beta'))
```

```suggestion
		if (version.contains('-'))
```
I would say we can change this too.